CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.0.6
+
+- Added new method to invalidate the purchaser info cache, useful when promotional purchases are granted from outside the app. #109
```

Comment:
maybe add an extra line here to separate a bit more from 3.0.5

Reply:
true!